src/fsharp/FSharp.Compiler.nuget/Microsoft.FSharp.Compiler.nuspec

```diff
@@ -40,13 +39,16 @@
         </dependencies>
     </metadata>
     <files>
-        <file src="fsc.exe"                           target="lib/netstandard1.6" />
-        <file src="FSharp.Compiler.dll"               target="lib/netstandard1.6" />
-        <file src="default.win32manifest"             target="runtimes/any/native/" />
-        <file src="FSharp.Build.dll"                  target="lib/netstandard1.6" />
-        <file src="Microsoft.FSharp.targets"          target="runtimes/any/native/" />
-        <file src="Microsoft.Portable.FSharp.targets" target="runtimes/any/native/" />
-        <file src="Microsoft.FSharp.NetSdk.targets"   target="runtimes/any/native/" />
-        <file src="Microsoft.FSharp.NetSdk.props"     target="runtimes/any/native/" />
+        <file src="coreclr/bin/fsc.exe"                           target="lib/netstandard1.6" />
```

Comment:
Though for that package the .NET Core version of the compiler binaries go in ``build\netcorepp1.0`` (but either way, not in ``lib`` https://github.com/fsharp/fsharp/blob/master/FSharp.Compiler.Tools.Nuget/FSharp.Compiler.Tools.nuspec#L67